valueformatting_test.go

```diff
@@ -267,6 +267,30 @@ func TestValueFormattingBinaryFormatter(t *testing.T) {
 	}
 }
 
+func TestValueFormattingJSONFormatter(t *testing.T) {
+	formatting := newValueFormatting()
+	formatter, err := formatting.jsonFormatter()
+
+	if err != nil {
+		t.Errorf("Error creating formatter: %v", err)
+	}
+
+	input := []byte("{\"name\": \"Brave\", \"age\": 2}")
+
+	want := `age:   2.00
+name: "Brave"`
+
+	got, err := formatter(input)
+	if err != nil {
+		t.Errorf("Error formatting JSON string: %v", err)
+	}
+
+	if !strings.Contains(got, want) {
+		t.Errorf("JSON not formatted correctly: wanted\n%s\n; got\n%s\n",
```

Comment:
%v is typically the preferred format specifier.

Reply:
Done.